components/common/layout/Header.js

```diff
@@ -64,26 +64,25 @@ class Header extends Component {
             <span />
           </Clickable>
         </div>
-        <div className={classNames('navbar-menu', { 'is-active': burgerActive })}>
-          <div className="navbar-start">
-            {/* // TODO: check permissions for routes */}
-            {NAVBAR_ROUTES.map(({ name, pattern = name, params, isActive }) => (
-              <Link key={name} route={name} params={params}>
-                <a className="navbar-item">
-                  <span
-                    className={classNames('rubric', { 'is-active': isActive(asPath, pattern) })}
-                  >
-                    <Text id={`header.${name}`} render={t => t.toUpperCase()} />
-                  </span>
-                </a>
-              </Link>
-            ))}
-          </div>
-
-          <div className="navbar-end">
-            <div className="navbar-item">
-              <LocaleContext.Consumer>
-                {lang => (
+        <LocaleContext.Consumer>
+          {lang => (
+            <div className={classNames('navbar-menu', { 'is-active': burgerActive })}>
+              <div className="navbar-start">
+                {/* // TODO: check permissions for routes */}
+                {NAVBAR_ROUTES.map(({ NavLink = Link, name, pattern = name, params, isActive }) => (
+                  <NavLink key={name} route={name} params={params} lang={lang}>
```

Comment:
Does it make sense to receive element ``<NavLink> ... </NavLink>`` from some function? I mean, code here is too nested